akka-actor/src/main/scala/akka/actor/ActorSystem.scala

```diff
@@ -57,6 +119,19 @@ object ActorSystem {
   def create(name: String): ActorSystem = apply(name)
 
   /**
+   * Java API: Creates a new actor system with the specified name and settings
+   * The core actor system settings are defined in [[ActorSystemBootstrapSettings]]
+   */
+  def create(name: String, settings: ActorSystemSettings): ActorSystem = apply(name, settings)
+
+  /**
+   * Java API: Shortcut for creating an actor system with custom bootstrap settings.
+   * Same behaviour as calling `ActorSystem.create(name, ActorSystemSettings.create(bootstrapSettings))`
+   */
+  def create(name: String, bootstrapSettings: ActorSystemBootstrapSettings): ActorSystem =
```

Comment:
Could you not dance around it with `settings && settings: ActorSystemSettings`? Like we do with attributes in streams I think